hack/make-rules/verify/all.sh

```diff
@@ -98,6 +98,12 @@ if [[ "${VERIFY_GO_DEPS:-true}" == "true" ]]; then
   hack/make-rules/verify/go-deps.sh || { FAILED+=($name); echo "ERROR: $name failed"; }
   cd "${REPO_ROOT}"
 fi
+if [[ "${VERIFY_CRIO_JOBS:-true}" == "true" ]]; then
```

Comment:
This is somewhat unusual, we mostly verify configs in a separate job / target and check repo-wide things here. It feels overly specific to have `make verify` cover this.

cc @chaodaiG any suggestions?

Reply:
Hm, how should we proceed with this?

Reply:
You could also just move this to a separate Make target (probably in a separate Makefile to not intrude on the current `global` flavor of the existing one).

Reply:
I removed the verification for now.